packages/mongodb-chatbot-ui/src/ModalView.tsx

```diff
@@ -89,16 +92,27 @@ export type ModalViewProps = ChatbotViewProps & {
 
 export function ModalView(props: ModalViewProps) {
   const { darkMode } = useDarkMode(props.darkMode);
-  const { initialMessageText, initialMessageSuggestedPrompts, showDisclaimer } =
-    props;
+  const {
+    disclaimer,
+    disclaimerHeading,
+    fatalErrorMessage = MongoDbChatbotFatalErrorMessage,
```

Comment:
```suggestion
    fatalErrorMessage = defaultChatbotFatalErrorMessage,
```

per above suggestion